RecoHI/HiTracking/python/hiMixedTripletStep_cff.py

```diff
@@ -1,206 +1,159 @@
+     precise = True,
+     useMultipleScattering = False,
+     beamSpot = "offlineBeamSpot",
+     useFoundVertices = True,
+     useFakeVertices       = False,
+     VertexCollection = "hiSelectedPixelVertex",
+     useFixedError = True,
+     fixedError = 3.75,#15.0
+     ptMin = 0.4,
+     originRadius = 1.0,
+     originRScaling4BigEvts = cms.bool(True),
+     halfLengthScaling4BigEvts = cms.bool(False),
+     ptMinScaling4BigEvts = cms.bool(True),
+     minOriginR = 0,
+     minHalfLength = 0,
+     maxPtMin = 0.7,
+     scalingStartNPix = 20000,
+     scalingEndNPix = 35000     
+))
+
+
+# seeding
+mixedTripletStepHitDoubletsB.clusterCheck = ""
+mixedTripletStepHitDoubletsB.trackingRegions = "hiMixedTripletStepTrackingRegionsB"
 
 # QUALITY CUTS DURING TRACK BUILDING
-import TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff
-hiMixedTripletTrajectoryFilter = TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff.CkfBaseTrajectoryFilter_block.clone(
-    maxLostHits = 0,
-    minimumNumberOfHits = 6,
-    minPt = 1.0
-    )
-
-# Propagator taking into account momentum uncertainty in multiple scattering calculation.
-import TrackingTools.MaterialEffects.MaterialPropagator_cfi
-hiMixedTripletPropagator = TrackingTools.MaterialEffects.MaterialPropagator_cfi.MaterialPropagator.clone(
-    ComponentName = 'hiMixedTripletPropagator',
-    ptMin = 1.0
-    )
-import TrackingTools.MaterialEffects.OppositeMaterialPropagator_cfi
-hiMixedTripletPropagatorOpposite = TrackingTools.MaterialEffects.OppositeMaterialPropagator_cfi.OppositeMaterialPropagator.clone(
-    ComponentName = 'hiMixedTripletPropagatorOpposite',
-    ptMin = 1.0
-    )
-
-import TrackingTools.KalmanUpdators.Chi2MeasurementEstimator_cfi
-hiMixedTripletChi2Est = TrackingTools.KalmanUpdators.Chi2MeasurementEstimator_cfi.Chi2MeasurementEstimator.clone(
-    ComponentName = cms.string('hiMixedTripletChi2Est'),
-    nSigma = cms.double(3.0),
-    MaxChi2 = cms.double(16.0)
-    )
-
+from RecoTracker.IterativeTracking.MixedTripletStep_cff import _mixedTripletStepTrajectoryFilterBase
+_mixedTripletStepTrajectoryFilterBase.minimumNumberOfHits = 5
+_mixedTripletStepTrajectoryFilterBase.minPt = 0.4
+mixedTripletStepPropagator.ptMin = 0.4
+mixedTripletStepPropagatorOpposite.ptMin = 0.4
 # TRACK BUILDING
-import RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi
-hiMixedTripletTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilder.clone(
-    MeasurementTrackerName = '',
-    trajectoryFilter = cms.PSet(refToPSet_ = cms.string('hiMixedTripletTrajectoryFilter')),
-    propagatorAlong = cms.string('hiMixedTripletPropagator'),
-    propagatorOpposite = cms.string('hiMixedTripletPropagatorOpposite'),
-    clustersToSkip = cms.InputTag('hiMixedTripletClusters'),
-    maxCand = 2,
-    estimator = cms.string('hiMixedTripletChi2Est')
-    )
+hiMixedTripletStepTracks = mixedTripletStepTracks.clone()
```

Comment:
please define tracks after the candidates just to have them in a natural sequence.